Identity/README.md

```diff
@@ -13,23 +7,12 @@ Import-Module ..\Connect\AzureStack.Connect.psm1
 Import-Module ..\Identity\AzureStack.Identity.psm1
 ```
 
-## Getting the directory tenant identifier from the Identity System
-
-This function is used to get the Directory Tenant Guid. This method works for both AAD and AD FS.
-
-```powershell
-$directoryTenantId = Get-AzsDirectoryTenantIdentifier -Authority "<DirectoryTenantUrl>"
-```
-
-An example of an authority for AAD is `https://login.windows.net/microsoft.onmicrosoft.com`
-and for AD FS is `https://adfs.local.azurestack.external/adfs`.
-
-## Creating a Service Principal in a disconnected (AD FS) topology
+# Creating a Service Principal in a disconnected (AD FS) topology
 
 You can create a Service Principal by executing the following command after importing the Identity module
 
 ```powershell
-$servicePrincipal = New-AzsAdGraphServicePrincipal -DisplayName "<YourServicePrincipalName>" -AdminCredential $(Get-Credential) -Verbose
+$servicePrincipal = New-AzsAdGraphServicePrincipal -DisplayName "<YourServicePrincipalName>" -AdminCredential $(Get-Credential) -ADFSMachineName "AZS-ADFS01" -Verbose
```

Comment:
Are we really exposing our internal machine names?   What is 01 is down... this seems like it shouldn't exist as a parameter